api/src/services/events.js

```diff
@@ -82,26 +80,18 @@ const verifySecret = async (id, secretKey) => {
  * @param {string} id
```

Comment:
This comment is really for line 74. `let event = await Event.findById(id);`  Mongoose doesn't return a promise here. It's a promise of a Query. We have to add `.exec()` at the end to get a promise for the event back.